### ISSUE TITLE
Hauptmenüpunkt "Projekte" umbenennen in "Mitmachen"

### ISSUE THREAD

**Issue by holgerd** [sozialhelden/wheelmap#396]
@Svenyo kannst Du das Wording bestätigen und den Englischen Begriff?

**Comment by 1000miles** [comment]
**Wording changes in the menu**
- [ ] Projekte --> Mitmachen 
- [ ] Menu "Projects" --> "Get involved" 

   => @Svenyo as far as we understood the changes need to be done on transifex first (resource files) before we can pull the changes for the navigation to the rails app. can you please check?  :)

----
Additionally to this task we will change following hardcoded URLs in the rails app as requested.

**Rails App**

- [x] http://news.wheelmap.org/botschafter-werden/ => http://news.wheelmap.org/wheelmap-botschafter/
- [x] http://news.wheelmap.org/en/become-a-wheelmap-ambassador/ => http://news.wheelmap.org/en/wheelmap-ambassador/

Related PR: #402

**Comment by Svenyo** [comment]
Hej @1000miles,
the main menu is not part of the language files in transifex. It has always only existed in German and English and no other languages. Please change directly.

**Comment by 1000miles** [comment]
@Svenyo 

If we change the translation for the key `projects` in the [EN relaunch.yml](https://github.com/sozialhelden/wheelmap/blob/02412ab4465790ed6ab1c63171561e0013c6aab9/config/locales/en/relaunch.yml#L32)

![00000158](https://cloud.githubusercontent.com/assets/4077916/18703046/959d99f0-7fe4-11e6-8f97-49ae9a62e47c.png)

The change will be visible in the header:

![00000157](https://cloud.githubusercontent.com/assets/4077916/18703133/e2f281ac-7fe4-11e6-8c11-2e4526c0ea67.png)

Basically we can change the translation for `Projects` to `Get involved` manually in all files, but the problem is that at the next pull of the latest transifex translations we still get the old translations for `projects`, if the origin source  (resource file) is not changed. 

It would be so great, if you could organize the translation of `projects` for all available languages so that we can pull the latest translations from transifex afterswards. :)

cc @holgerd

**Comment by Svenyo** [comment]
Ok, I found the corresponding text string in relaunch.yml on transifex.
We will include it in the next language update. I'll try to finalize the translations till friday.

@1000miles @holgerd

**Comment by Svenyo** [comment]
@1000miles cc @holgerd The string in the navigation has been changed into "Get involved" and "Mitmachen" and is now available in the updated language file in Transifex.

**Comment by 1000miles** [comment]
@Svenyo cc @holgerd & @karolinamakaruk 

The PR #475 that resolved this issue (#396) & #479 has been deployed on staging under release [staging-20161125132725](https://github.com/sozialhelden/wheelmap/releases/tag/staging-20161125132725) today. It can now be reviewed.

![locale_switcher_nav_spanish](https://cloud.githubusercontent.com/assets/4077916/20628178/d3479706-b324-11e6-8c55-9d60ca8123f9.png)

**Comment by karolinamakaruk** [comment]
@1000miles I tested it. I changed languages and 'get involved' is now displayed in different languages on https://staging.wheelmap.org/pl/map#/?zoom=14 .

**Comment by 1000miles** [comment]
Merci. We will deploy this on production tomorrow.

**Comment by 1000miles** [comment]
Closed. Deployed on production with release: https://github.com/sozialhelden/wheelmap/releases/tag/production-20161129150628.
